tests/test_lint.py

```diff
@@ -266,15 +279,20 @@ def test_conda_env_fail(self):
         expectations = {"failed": 3, "warned": 1, "passed": 2}
         self.assess_lint_status(lint_obj, **expectations)
 
-    def test_conda_env_timeout(self):
+    @mock.patch('requests.get')
+    def test_conda_env_timeout(self, mock_get):
         """ Tests the conda environment handles API timeouts """
+        # Define the behaviour of the request get mock
+        timeout = requests.exceptions.Timeout()
+        mock_get.side_effect = timeout
+        # Now do the test
         lint_obj = nf_core.lint.PipelineLint(PATH_WORKING_EXAMPLE)
         lint_obj.files = ['environment.yml']
         with open(os.path.join(PATH_WORKING_EXAMPLE, 'environment.yml'), 'r') as fh:
             lint_obj.conda_config = yaml.load(fh)
         lint_obj.pipeline_name = 'tools'
-        lint_obj.check_conda_env_yaml(api_timeout=0.0001)
```

Comment:
This is much nicer 👍 Can we remove the `api_timeout` parameter from that function completely now and just hardcode the value instead? I don't think we actually need it apart from for this test.

Reply:
yup, agree!